libraries/AP_Scripting/tests/luacheck.lua

```diff
@@ -1,12 +1,57 @@
 
--- Don't check globals yet. This requires us to add a list of all the AP bindings
-global = false
-
 -- https://luacheck.readthedocs.io/en/stable/warnings.html
-ignore = {"631", -- Line is too long
+ignore = {"111", -- Setting an undefined global variable.
+          "113", -- Accessing an undefined global variable.
```

Comment:
Have to add these two otherwise EVERYTHING in the script has to be `local`. Including the update function (scripts still run like this, so we could enable the check, but it seems a bit much)